src/utils/gh-auth.js

```diff
@@ -108,15 +113,19 @@ const authWithToken = async () => {
   throw error
 }
 
+/**
+ * Get a github token
+ * @returns {Promise<Token>} Returns a Promise with a token object
+ */
 const getGitHubToken = async () => {
   log('')
 
   const withNetlify = await promptForAuthMethod()
   if (withNetlify) {
-    return await authWithNetlify()
+    return authWithNetlify()
```

Comment:
See [here](https://github.com/goldbergyoni/nodebestpractices#-212-always-await-promises-before-returning-to-avoid-a-partial-stacktrace) on the reason we use `return await`

Reply:
Thx have not known about the code guidelines 👍  will read through :)

Reply:
Yes, this should be a lint rule actually (e.g. the opposite of https://eslint.org/docs/rules/no-return-await)

Reply:
Update: I wonder why https://eslint.org/docs/rules/require-await didn't catch this. We have it [configured](https://github.com/netlify/eslint-config-node/blob/5dd59a59882b0a9cee04e376d3126a72da036daa/.eslintrc.js#L126)